starlark/eval_test.go

```diff
@@ -35,6 +35,12 @@ func option(chunk, name string) bool {
 	return strings.Contains(chunk, "option:"+name)
 }
 
+// Copy of the xerrors.Wrapper interface
```

Comment:
// Wrapper is the type of errors with an Unwrap method; see https://golang.org/pkg/errors.

Reply:
done